content/docs/user-guide/managing-external-data.md

```diff
@@ -33,25 +29,26 @@ Currently, the following types (protocols) of external outputs (and
 - HDFS
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
+> Note [remote storage](/doc/command-reference/remote) is a separate feature,
+> and that external outputs are not pushed or pulled from/to DVC remotes.
 
-> Avoid using the same [DVC remote](/doc/command-reference/remote) (used for
-> `dvc push`, `dvc pull`, etc.) for external outputs, because it may cause file
-> hash overlaps: the hash of an external output could collide with a hash
-> generated locally for another file with different content.
+Importantly, external outputs require an
```

Comment:
I'm a bit lost here ... I think the previous version was pretty precise, why change it?

Reply:
> why change it?

The previous version was:

> For cached external outputs (e.g. dvc add, dvc run -o), you will need to setup an external cache in the same external/remote file system first.

We decided to not mentioned "cached outs" here since there's no real use case for non cached external outs. The mention of `add` and `run -o` also don't seem that important. So most of that sentence had to go anyway. Then we decided to incorporate the info involving remotes to the paragraphs before the bullet list so pretty much all the text changed. And it was a good thing, I think, because it explains what ext outs are first, emphasizes in dvc.yaml, and how their URLs are formed (same as remotes).

Reply:
I removed "Importantly" now, and moved this paragraph before the note about remotes being a different feature (right after the bullet list). I think the structure of this section makes sense: 1 what are ext outs, 2 how to define them, 3 note about external cache.